drivers/haptics/Kconfig.drv2605

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Cirrus Logic, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+config HAPTICS_DRV2605
+	bool "DRV2605 Haptics Driver"
+	default y
+	depends on DT_HAS_TI_DRV2605_ENABLED
+	select I2C
+	help
+	  Enable I2C-based driver for the Texas Instruments DRV2605 Haptics
+	  Driver.
+
+config HAPTICS_DRV2605_INIT_PRIORITY
+	int "DRV2605 haptic device driver init priority"
+	default KERNEL_INIT_PRIORITY_DEVICE
+	depends on HAPTICS_DRV2605
+	help
+	  Init priority for the Texas Instruments DRV2605 Haptic device driver.
```

Comment:
Just use the HAPTICS_INIT_PRIORITY in the device driver :) Additionally, to add driver specific configs, hide them behind
```
if HAPTICS_DRV2605
...
endif
```
rather than `depends on HAPTICS_DRV2605`

Reply:
ack, I'll just use `HAPTICS_INIT_PRIORITY` :)